tools/slack_tools/collect_prs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+. tools/shell_utils.sh
```

Comment:
im very much trying to get rid of this - can we use bazel for dependencies?

Reply:
Do we have examples of scripts that run that way?  my tooling skills are pretty weak so I just copied this from one of the other scrips

Reply:
i can follow up after and shift this to bazel if thats helpful

Reply:
Ultimately we are going to run this as a GH action. I think it will be much easier if we just stick with raw pip for this tool since I think it will be trivial to get running in GHA. So I think the end result here is we will lose the SH file and just keep the requirements.txt file. I will follow up once I do a template for GHA.

Reply:
if we are using gh actions to trigger this we probs want to use something like this i think https://github.com/marketplace/actions/pip-installer